Week1/cash.c

```diff
@@ -0,0 +1,25 @@
+#include <cs50.h>
+#include <stdio.h>
+
+void cashback();
```

Comment:
Declaration of function is wrong; please favour explicit declaration over implicit one:

```suggestion
void cashback(int);
```